weblate/trans/models/component.py

```diff
@@ -838,6 +853,9 @@ def save(self, *args, **kwargs) -> None:
             )
```

Comment:
Missing condition here:

```suggestion
                or old.key_filter != self.key_filter
            )
```

Reply:
Added this condition.